.devcontainer/Dockerfile

```diff
@@ -0,0 +1,5 @@
+ARG VARIANT=20-bullseye
+FROM mcr.microsoft.com/vscode/devcontainers/typescript-node:0-${VARIANT}
+
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+    && apt-get -y install --no-install-recommends postgresql-client 
```

Comment:
Most of the files in the repo have a final newline, so I think for consistency it should be added here, too.

Reply:
Fixed, but curious why it matters.  Sorry, not trying to ague, just curious.  Any reason?

Reply:
Fixed, but curious why it matters.  Sorry, not trying to ague, just curious.  Any reason?

Reply:
Disclaimer: I'm a passer-by, not a maintainer.

I personally treat consistent EOLs simply as a part of general linting. This is similar to how we treat ending semicolons - the language allows to have them or omit them; but we stick to consistency for some reason. Also, in case of Github, the large red stop signs in diff viewer a bit repulsive.